ui/components/multichain/import-nfts-modal/import-nfts-modal.js

```diff
@@ -47,10 +47,10 @@ import {
   IconSize,
   Label,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import Tooltip from '../../ui/tooltip';
 import { useNftsCollections } from '../../../hooks/useNftsCollections';
 import { checkTokenIdExists } from '../../../helpers/utils/util';
```

Comment:
Screenshots showing no visual changes to UI

### Before

<img width="1512" alt="Screenshot 2023-12-10 at 3 43 39 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/06769a04-4de0-47f1-a845-4524757fd033">

### After

<img width="1512" alt="Screenshot 2023-12-10 at 3 43 50 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/479cb83c-5a7f-408d-b067-4562d22d7183">